modules/ROOT/pages/install-aws.adoc

```diff
@@ -122,7 +122,6 @@ Refer to xref:manage-index.adoc[Manage Runtime Fabric] for detailed information.
 *Required Environment Variables*
 
 [%header,cols="3*a"]
-.Required Environment Variables
```

Comment:
we should still have the required variables section?

Reply:
And perhaps just add disable_selinux to optional variables section? @sarjeet2013?

Reply:
I'm removing the table titles, not the headings.